modules/swagger-codegen/src/main/resources/Java/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Smooch Technologies Inc.
```

Comment:
should the dates be updated?

Reply:
Sent an email to opensource@zendesk.com asking what the company name should be here. We may need to use a different license altogether as indicated here: https://zendesk.atlassian.net/wiki/spaces/ENG/pages/32244558/How+to+license+a+project